### PR TITLE
update count.js to use the sqlite db

### PR DESCRIPTION
**Pull request by dcepil**
updated `count.js`:
- now the command saves/reads the count amount to/from the database instead of `.json` files

### CODE REVIEW

**Comment by dcepil** [comment]
[Update !count to use the database](https://trello.com/c/dzOpvm2I/24-update-count-to-use-the-database)

**Comment by dcepil** [comment]
[Update !count to use the database](https://trello.com/c/dzOpvm2I/24-update-count-to-use-the-database)

**Comment by dcepil** [comment]
[Update !count to use the database](https://trello.com/c/dzOpvm2I/24-update-count-to-use-the-database)

**Comment by dcepil** [comment]
[Update !count to use the database](https://trello.com/c/dzOpvm2I/24-update-count-to-use-the-database)

**Comment by dcepil** [comment]
[Update !count to use the database](https://trello.com/c/dzOpvm2I/24-update-count-to-use-the-database)

**Comment by dcepil** [comment]
[Update !count to use the database](https://trello.com/c/dzOpvm2I/24-update-count-to-use-the-database)

**Comment by dcepil** [comment]
[Update !count to use the database](https://trello.com/c/dzOpvm2I/24-update-count-to-use-the-database)

File: commands/count.js [hunk 1]
```diff
@@ -18,25 +18,23 @@ const embed = (msg, args, count, userCount) => {
...
-    if (user.count) {
-      user.count += count;
-    } else {
-      user.count = count;
-    }
-  }
-  fs.writeFileSync(filePath, JSON.stringify(user));
+  user.count += count;
+  await user.save().catch((err) => {
+    console.err(err);
```
Comment (VuongRon) [inline commands/count.js]: May want to return here if error

Comment (dcepil) [reply commands/count.js]: Done.

**Review by VuongRon — APPROVED** [review]
Amazing

**Pull request merged**